Assets/SC_Attackforenemyclassobjects.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SC_Attackforenemyclassobjects : MonoBehaviour
+{
+    private Transform _target;
+    private bool _isReadyToAttackThePlayer; //var to show if he is ready to attack the player
+    [SerializeField] private float attackRate; //how fast an attack 
+
+    private EnemyConfig _enemyConfig; //no idea where that class is
```

Comment:
This class does not exist, you should remove its use because the code will cause an error.